Controller/FormController.php

```diff
@@ -174,6 +175,50 @@ public function postAction(Request $request): Response
         return $this->handleView($this->view($this->getApiEntity($entity, $locale), 201));
     }
 
+    public function postTriggerAction(Request $request, int $id): Response
+    {
+        $action = $request->query->get('action');
+        $locale = $this->getLocale($request);
+        $entity = $this->formManager->findById($id, $locale);
+
+        if (!$entity) {
+            throw new NotFoundHttpException(sprintf('No form with id "%s" was found!', $id));
+        }
+
+        try {
+            switch ($action) {
+                case 'copy':
+                    $copiedForm = null;
+                    foreach ($entity->getTranslations() as $translation) {
+                        /** @var array{ title: string, fields: mixed[] } $data */
+                        $data = $this->getApiEntity($entity, $translation->getLocale());
+                        $data['title'] = $data['title'] . ' (2)';
+
+                        /** @var array{ options: \stdClass|mixed[] } $field */
+                        foreach ($data['fields'] as &$field) {
```

Comment:
this is needed because the `getApiEntity` method returns an empty `\stdClass` here: https://github.com/sulu/SuluFormBundle/blob/c3908bd532df7563759a6fcc270f762e9e1d40f9/Controller/FormController.php#L367